spark/dl/src/main/scala/com/intel/analytics/bigdl/models/utils/ModelBroadcast.scala

```diff
@@ -141,13 +173,92 @@ class ModelBroadcast[T: ClassTag](applyProtoBuffer: Boolean = false)
       Array()
     }
   }
-
 }
 
-
 object ModelBroadcast {
   def apply[@specialized(Float, Double) T: ClassTag](applyProtoBuffer: Boolean = false)
         (implicit ev: TensorNumeric[T]) : ModelBroadcast[T] = {
     new ModelBroadcast(applyProtoBuffer)
   }
 }
+
+class ModelInfo[T: ClassTag](val uuid: String, @transient var model: Module[T])(
```

Comment:
case class ?

Reply:
Why transient ? will it be ignored while serialized and deserialized between driver and executor ?